pkg/apply/task/inv_set_task.go

```diff
@@ -124,3 +128,36 @@ func (i *InvSetTask) Cancel(_ *taskrunner.TaskContext) {}
 
 // StatusUpdate is not supported by the InvSetTask.
 func (i *InvSetTask) StatusUpdate(_ *taskrunner.TaskContext, _ object.ObjMetadata) {}
+
+func (i InvSetTask) getApplyReoncileStatus(taskContext *taskrunner.TaskContext) map[object.ObjMetadata]status.ApplyReconcileStatus {
```

Comment:
This function will probably not make sense to any future code reader. We should add a significant comment to this function describing which objects are being returned (e.g. objects we've computed in the cluster which belong in the inventory).

Reply:
Added some comments for this function.